shelley/pool-ranking/pool-ranking.tex

```diff
@@ -28,78 +29,119 @@
+%
+We represent such an estimate as a probability distribution on the interval $[0,1]$.\todo{Are you assuming a Gaussian distribution?}
 
 After each epoch, we refine our estimate into a more accurate one as follows.
-
-Let's say we
-observe the output of a stake pool during an epoch. We have a previous estimate, $w$, the
-epoch has $E$ slots, the pool produces $n$ blocks, and we calculate that on a given slot
-the chance they're allowed to produce a block is $t$.
-
-We compute the likelihood function, $L$, where $L(x)$ is the probability of producing exactly $n$ blocks,
+%
+Assume that we can
+observe all the blocks that a stake pool produces during an epoch, that we have a previous estimate of the blocks that
+we expected to produce during this epoch, $w$; we know that the
+epoch has a total of $E$ slots; the pool produces $n$ blocks; and we calculate that on a given slot
+the chance that the pool is allowed to produce a block is $t$.
+%
+The \emph{likelihood function}, $L$, is then shown below, where $L(x)$ is the probability of producing exactly $n$ blocks,
+and $S(x)$ is the probability that the pool produces a block at a given slot, under the assumption that the
+\emph{hit rate}, $p$, is equal to $x$.
 
 \[ L(x) = \left( E \atop n \right) S(x)^{n} (1-S(x))^{E - n}, \]
 
-and $S(x)$ is the probability that the pool produces a block at a given slot under the assumption that the
-hit rate, $p$, is equal to $x$.
-
 \[
 \begin{array}{r@{=}l}
 S(x) & P( \text{made block} | p = x) \\
-     & t \cdot P( \text{made block} | p = x,~\text{can}) + (1-t) \cdot P( \text{made block} | p = x,~\text{can't}) \\
+     & t \cdot P( \text{made block} | p = x,~b) + (1-t) \cdot P( \text{made block} | p = x,~\not~b) \\
+%     & t \cdot P( \text{made block} | p = x,~\text{can}) + (1-t) \cdot P( \text{made block} | p = x,~\text{can't}) \\
      & tx                              + (1-t) \cdot 0 \\
      & tx \\
+\\
+b    & \text{can make a block}
 \end{array}
 \]
 
-Our refined estimate is then $q(x) = \frac{w(x)L(x)}{\int w(x)L(x)}$.
+Our refined hit rate estimate is then $q(x) = \frac{w(x)L(x)}{\int w(x)L(x)}$.
 
-By calculating a running product, $\mathcal{L}$, of likelihood functions, we can defer this computation.
-
-At the time that a user wishes to produce a ranking, they will
-supply a prior distribution. This is a default estimate applied to pools for which we don't have any historical data.
-
-To make the rankings we use the posterior distribution,
+\todo{Needs a better explanation. The equation implies that $L = \mathcal{L}$}
+In order to obtain an actual ranking, we maintain a running product, $\mathcal{L}$, of the likelihood function, $L$.
+We apply this to some initial \emph{prior distribution}
+of block production, where % , we can defer the full computation  until we need to use it.
+% When a user wishes to produce a ranking, they will
+% supply a \emph{prior distribution} of block production that can be plugged in to this function.
```

Comment:
I think it's nice to note that we can defer coming up with a prior until the very last second. this mean that deadalus could, in theory, let the user have some say in what prior is used to give them their ranking.